dashboard/config/scripts_json/coursea-2021.script_json

```diff
@@ -14,23 +14,13 @@
       "lesson_extras_available": true,
       "version_year": "2021",
       "curriculum_umbrella": "CSF",
-      "teacher_resources": [
-        [
-          "teacherForum",
-          "http://forum.code.org/c/csf"
-        ],
-        [
-          "curriculumGuide",
-          "https://docs.google.com/document/d/1hstUHTGIdvtPP0TDdEfQeXG3zQ0q6ys237n2BY9CQmg/preview"
-        ]
-      ],
       "is_course": true,
       "is_migrated": true,
       "has_lesson_plan": true
     },
     "new_name": null,
     "family_name": "coursea",
-    "serialized_at": "2021-03-24 21:38:25 UTC",
+    "serialized_at": "2021-03-30 16:02:40 UTC",
```

Comment:
I'd recommend removing these `serialized_at` lines from your diffs, since they have a high chance of merge conflict, but it's up to you to do whatever lets you get this merged in the easiest

Reply:
Yeah, that's definitely part of the merge conflicts that we're seeing. I'll remove them when I regenerate the diffs